dockerfiles/Dockerfile.dev

```diff
@@ -47,7 +47,7 @@ RUN  apk add --no-cache \
 RUN echo -e "\nYou are now in a development container. Run '\e\033[1mmake help\e\033[0m' to learn about\navailable make targets.\n" > /etc/motd \
  && echo -e "cat /etc/motd\nPS1=\"\e[0;32m\u@docker-cli-dev\\$ \e[0m\"" >> /root/.bashrc \
  && echo -e "source /etc/bash/bash_completion.sh" >> /root/.bashrc
-CMD bash
+CMD ["bash"]
```

Comment:
This should probably be `/bin/bash`;

```bash
docker run -it --rm alpine
ask add --no-cache bash

command -v bash
/bin/bash
```

Reply:
Thanks! Looks like the second commit is missing a DCO  sign-off, which  makes CI fail


could you squash both commits (we prefer not  to keep history for  fix-ups like this), and amend the commit message so that it has a DCO sign-off?

Let me know if you need help doing so!